translate/src/context/MachineryTranslations.tsx

```diff
@@ -99,15 +99,24 @@ export function MachineryProvider({
 
       // Only make requests to paid services if user is authenticated
       if (isAuthenticated) {
-        if (locale.googleTranslateCode) {
+        const root = document.getElementById('root');
+
+        const isGoogleTranslateSupported =
+          root?.dataset.isGoogleTranslateSupported === 'true';
```

Comment:
Probably better to use `JSON.parse()`? It'll be safer, but the there will be a bit more code.

Reply:
No, this is better